sdk/ai/azure-ai-resources/azure/ai/resources/client/_ai_client.py

```diff
@@ -257,7 +257,7 @@ def build_index_on_cloud(
         embeddings_model="text-embedding-ada-002",
         aoai_connection_id: str = DEFAULT_OPEN_AI_CONNECTION_NAME,
         ######## ACS index info ########
-        acs_config: ACSOutputConfig = None,  # todo better name?
+        index_config: Union[ACSOutputConfig, PineconeOutputConfig] = None,  # todo better name?
```

Comment:
Same as in azure-ai-generative, would this be considered breaking existing functionality?